bpf/include/bpf/helpers_skb.h

```diff
@@ -54,9 +54,6 @@ static int BPF_FUNC_REMAP(skb_event_output, struct __sk_buff *skb, void *map,
 static struct bpf_sock *BPF_FUNC(skc_lookup_tcp, struct __sk_buff *skb,
 				 struct bpf_sock_tuple *tuple, __u32 tuple_size,
 				 __u64 netns, __u64 flags);
-static struct bpf_sock *BPF_FUNC(sk_lookup_udp, struct __sk_buff *skb,
```

Comment:
Why is this helper being deleted from the `helpers_skb.h`?

Reply:
Somehow I was misinterpreting the change to be introducing specific helpers for SK rather than sharing the same helper everywhere 🤦. I was also confused by how one of the socket helpers remains here but it's because that one is not used more generally, only in the skb contexts. So all is fine here, sorry for the noise.